src/stores/AccountController.ts

```diff
@@ -389,6 +391,34 @@ export async function reloadActiveSubscription({ delay }: { delay: number } = {
   });
 }
 
+/**
+ * Get multiple media items for the given IDs. This performs multiple GET requests and optionally fetches the tokens
+ * needed when URL signing (with DRM) is enabled.
+ *
+ * @todo This function is temporarily until we add support for the watchlist playlist
+ * {@link https://github.com/jwplayer/ott-web-app/issues/43}. We could also refactor the favorites and watch history
+ * playlist to use the `usePlaylist` hook as well.
+ *
+ * @param mediaIds
+ */
+export async function getMediaItems(mediaIds: string[]) {
+  const jwt = useAccountStore.getState().auth?.jwt;
+  const signingConfig = useConfigStore.getState().config.contentSigningService;
+  const host = signingConfig?.host;
+  const drmPolicyId = signingConfig?.drmPolicyId;
+  const drmEnabled = !!drmPolicyId && !!signingConfig?.drmEnabled;
+  const signingEnabled = !!host && drmEnabled && !!drmPolicyId;
+
+  if (signingEnabled) {
```

Comment:
Is it not possible to have signing enabled without DRM?

Reply:
See the comment above, this call is only used for UI, not for video playback.